### PR TITLE
Redirect sublicense and approved usages to www.jenkins.io

### PR DESCRIPTION
**Pull request by MarkEWaite**
## Redirect sublicense and approved usages to www.jenkins.io

Needs https://github.com/jenkins-infra/jenkins.io/pull/3869 before this is merged.

### CODE REVIEW

**Review by markjacksonfishing — APPROVED** [review]

**Review by olblak — APPROVED** [review]

**Review by olblak — CHANGES_REQUESTED** [review]
Both redirection redirect to pages that do not exist on jenkins.io

**Comment by olblak** [comment]
Also it would be nice to start using 'www.jenkins.io' instead of 'jenkins.io'

**Comment by MarkEWaite** [comment]
@olblak the final destination pages were merged as part of https://github.com/jenkins-infra/jenkins.io/pull/3869 .  I've updated the pull request to use www.jenkins.io and the final destination URLs.

When this is merged, please delete the branch `redirect-sublicense-and-trademark-usages`.  It was created inside this repository instead of being created in my fork.  My mistake for using `hub pull-request -p`.  That caused it to submit the branch to upstream and then create a pull request, instead of submitting the branch to my fork and then creating the pull request.

**Review by olblak — APPROVED** [review]

**Comment by olblak** [comment]
@MarkEWaite  It appears that the vhost check doesn't pass

File: dist/profile/templates/confluence/vhost.conf [hunk 1]
```diff
@@ -1614,6 +1614,8 @@ RewriteRule "^/display/JENKINS/Board\+Election\+Process$" "https://jenkins.io/pr
 RewriteRule "^/display/JENKINS/Governance\+Board$" "https://jenkins.io/project/board/" [NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Governance\+Document$" "https://jenkins.io/project/governance/" [NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Governance\+Meeting\+Agenda$" "https://jenkins.io/project/governance-meeting/" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Approved\+Trademark\+Usage$" "https://www.jenkins.io/project/trademark/approved-usage/" [NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Trademark+Sublicense$" "https://www.jenkins.io/project/trademark/sublicense/" [NC,L,QSA,R=301]
```
Comment (oleg-nenashev) [inline dist/profile/templates/confluence/vhost.conf]:
```suggestion
RewriteRule "^/display/JENKINS/Trademark\+Sublicense$" "https://www.jenkins.io/project/trademark/sublicense/" [NC,L,QSA,R=301]
```

**Review by olblak — APPROVED** [review]

**Pull request merged**